core/ir/aarch64/codec.c

```diff
@@ -226,8 +226,11 @@ encode_pc_off(OUT uint *poff, int bits, byte *pc, instr_t *instr, opnd_t opnd,
         *poff = off >> 2 & (range - 1);
         return true;
     }
-    /* If !di->check_reachable we still require correct alignment. */
-    if (!di->check_reachable && ALIGNED(off, 4)) {
+    /* If !di->check_reachable we do not require correct alignment as there is
+     * a common use case of a label instruction operand whose note value holds
```

Comment:
The note field should be interpreted as a pc-relative offset only for non-label instructions right? So, how about skipping the alignment check if `instr_is_label(instr)`?

Reply:
No, the note field holds the offset from the start of the instruction list for all instructions, including and especially labels as they are often targets of jumps and thus we need to know where they live.

Maybe what you mean is to keep the alignment check for `opnd.kind == PC_kind`?  That seems reasonable.  Done.